islandora_transforms/BPL_MODS_to_solr.xslt

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- BPL requires some indexing of cases not handled by standard MODS slurping -->
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:foxml="info:fedora/fedora-system:def/foxml#"
+  xmlns:mods="http://www.loc.gov/mods/v3"
+     exclude-result-prefixes="mods">
+
+  <xsl:template match="mods:mods[1]" mode="slurp_for_bpl">
```

Comment:
This template doesn't have an entry point? As in, we never hit the mode `slurp_for_bpl`?